zerver/tests/test_message_move_topic.py

```diff
@@ -106,6 +106,44 @@ def test_edit_message_no_topic(self) -> None:
         )
         self.assert_json_error(result, "Topic can't be empty!")
 
+    def test_topic_required_in_mandatory_topic_realm(self) -> None:
+        self.login("iago")
+        admin_user = self.example_user("iago")
+        hamlet = self.example_user("hamlet")
+        realm = admin_user.realm
+        realm.mandatory_topics = True
```

Comment:
It would be better to set this by calling `do_set_realm_property`.

Reply:
Done!